CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+### ServiceNow Open Source Code-of-Conduct
```

Comment:
Some of these don't make much sense here, I'm assuming t's servicenow boilerplate?

Reply:
We need to ask Sean what needs to be included. Got all of this from Azimuth